<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cmbRenk">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>darkCyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>31</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Arka Plan Rengini Seçiniz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRenk">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seçtiğiniz Renk:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHatirla">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Hatırla</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDemirbas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDemirbasEkle">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Demirbaş Ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKayitSil">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydı Sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTemizle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Listeyi Temizle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTamam">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIptal">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listDemirbas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>491</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
